index.js

```diff
@@ -175,6 +187,28 @@ const parseArgs = (
   return result;
 };
 
+// Look ahead to next token in argv array:
+function peek(argv, pos) {
+  if (pos + 1 >= argv.length) {
+    return 0;
+  }
+  return token(argv[pos + 1]);
+}
+
+// Guess token type based on individual string from argv:
+function token(arg) {
+  const chr = StringPrototypeCharAt(arg, 0);
+  const nextChr = StringPrototypeCharAt(arg, 1);
+  if (StringPrototypeMatch(chr, /^[0-9]$/)) {
+    return TOKENS.NUMERIC;
+  } else if (chr === '-' && StringPrototypeMatch(nextChr, /^[0-9]$/)) {
+    return TOKENS.NUMERIC;
+  } else if (chr === '-') {
+    return TOKENS.FLAG;
```

Comment:
Two other things we could identify using `token()` are `-` and `--`. But the focus of this PR is negative numbers, so leave that for future refactor.